pkg/components/components.go

```diff
@@ -301,24 +296,128 @@ func GetClusterRole() *rbacv1.ClusterRole {
 				},
 				Resources: []string{
 					"networkaddonsconfigs",
+					"networkaddonsconfigs/status",
 				},
 				Verbs: []string{
 					"get",
 					"list",
 					"watch",
+					"update",
 				},
 			},
 			{
 				APIGroups: []string{
+					"",
+				},
+				Resources: []string{
+					"services",
+					"configmaps",
+					"namespaces",
+					"nodes",
+					"nodes/status",
+					"serviceaccounts",
+					"pods",
+					"pods/status",
+					"events",
+					"secrets",
+				},
+				Verbs: []string{
 					"*",
 				},
+			},
+			{
+				APIGroups: []string{
+					"apps",
+				},
+				Resources: []string{
+					"*",
+				},
+				Verbs: []string{
+					"*",
+				},
+			},
+			{
+				APIGroups: []string{
+					"rbac.authorization.k8s.io",
+				},
+				Resources: []string{
+					"*",
+				},
+				Verbs: []string{
+					"*",
+				},
+			},
+			{
+				APIGroups: []string{
+					"admissionregistration.k8s.io",
+				},
+				Resources: []string{
+					"mutatingwebhookconfigurations",
+					"validatingwebhookconfigurations",
+				},
+				Verbs: []string{
+					"*",
+				},
+			},
+			{
+				APIGroups: []string{
+					"k8s.cni.cncf.io",
```

Comment:
This is for NAD, right? We are not creating any network attachment definition from the operator